patches/api/0357-Multi-Block-Change-API.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Brody Beckwith <brody@beckwith.dev>
+Date: Fri, 14 Jan 2022 00:40:42 -0500
+Subject: [PATCH] Multi Block Change API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 7eda2ba17e39b8183e572c1cefa8afffbf17afcb..2c7a8e03b04ec176f4738a2923cd3e15b5f2b523 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1,6 +1,7 @@
+ package org.bukkit.entity;
+ 
+ import java.net.InetSocketAddress;
++import java.util.Map; // Paper
+ import java.util.UUID;
+ import com.destroystokyo.paper.ClientOption; // Paper
+ import com.destroystokyo.paper.Title; // Paper
+@@ -584,6 +585,17 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     public void sendBlockDamage(@NotNull Location loc, float progress);
+ 
++    // Paper start
++    /**
++     * Send multiple block changes. This fakes a multi block change packet for each
++     * chunk section that a block change occurs. This will not actually change the world in any way.
++     *
++     * @param blockChanges A map of the locations you want to change to their new block data
++     * @param suppressLightUpdates Whether to suppress light updates or not
++     */
++    public void sendMultiBlockChange(@NotNull Map<Location, BlockData> blockChanges, boolean suppressLightUpdates);
```

Comment:
Another default method calling `sendMultiBlockChange(blockChanges, false)` would be nice

Reply:
Added.